<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TP4Class</class>
 <widget class="QMainWindow" name="TP4Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>541</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionInscrire_Candidat"/>
    <addaction name="separator"/>
    <addaction name="actionInscrire_Electeur"/>
    <addaction name="separator"/>
    <addaction name="actionDesinscription"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuOperations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInscrire_Candidat">
   <property name="text">
    <string>Inscrire Candidat</string>
   </property>
  </action>
  <action name="actionInscrire_Electeur">
   <property name="text">
    <string>Inscrire Electeur</string>
   </property>
  </action>
  <action name="actionDesinscription">
   <property name="text">
    <string>Desinscription</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>TP4Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
